tests/contrib/grpc/hello_pb2_grpc.py

```diff
@@ -16,12 +16,22 @@ def __init__(self, channel):
       channel: A grpc.Channel.
     """
     self.SayHello = channel.unary_unary(
-        '/Hello/SayHello',
```

Comment:
Like this, we don't have `/package.service/method`

Reply:
The `package` is defined in the `hello.proto` but to your point I've added a test for the utility function in `test_grpc_utils.py`.